src/randindex.jl

```diff
@@ -0,0 +1,59 @@
+function contingency(mem1,mem2)
+	# Form contigency matrix for two vectors
+	# contigency = contigency(mem1,mem2) returns contingency matrix for two
+	# vectors mem1, mem2. These define which cluster each entity has been assigned to.
+	#
+
+	contigency = zeros(maximum(mem1),maximum(mem2))
+
+	for i = 1:length(mem1)
+	   contigency[mem1[i],mem2[i]] += 1
+	end
+
+	return contigency
+end
```

Comment:
Should this be replaced by a call to StatsBase.counts? It is already imported above.